### PR TITLE
HPCC4J-523 Incorrect Java version linking based on JDK

### PR DESCRIPTION
**Pull request by jpmcmu**
- Updated POM file to use release configuration
- Updated Github actions to use JDK 11 required by release build feature

Signed-off-by: James McMullan James.McMullan@lexisnexis.com

### CODE REVIEW

**Comment by jpmcmu** [comment]
@rpastrana please review. Note: failure is due to old test suite issues, will be fixed by HPCC4j-512 PR.

File: DataAccess/pom.xml [hunk 1]
```diff
@@ -84,6 +84,13 @@
   <build>
     <pluginManagement>
       <plugins>
+        <plugin>
```
Comment (rpastrana) [inline DataAccess/pom.xml:87]: Why did we end up having to declare the maven-compiler-plugin multiple times? Is there a way to apply it globally? If not, is there a well placed location for a comment reminding future developers to include this plugin in any new profiles, build, etc sections?

**Review by rpastrana — COMMENTED** [review]
@jpmcmu looks good, just one comment/suggestion

File: DataAccess/pom.xml (hunk 1 above)
Comment (jpmcmu) [reply DataAccess/pom.xml:87]: Yep, there is a way to have that declared only once I will do that, and then squash.

**Comment by jpmcmu** [comment]
@rpastrana Removed the un-needed plugin settings in the profiles. Need to keep the plugin settings in pluginManagement & plugins though.

**Pull request merged**